tests/cmdline/commands/test_export.py

```diff
@@ -181,18 +181,6 @@ def test_migrate_version_specific(self):
         finally:
             delete_temporary_file(filename_output)
 
-    def test_migrate_versions_recent(self):
```

Comment:
Related to my other comment about changing the migration functionality behaviour.